templates/partners/desktop.html

```diff
@@ -0,0 +1,196 @@
+        </div>
+        <div class="col-6 u-align--center u-hide--small">
+          <img src="https://assets.ubuntu.com/v1/690d0479-disco-dingo-laptop.png" alt="charmed kubernetes image">
+        </div>
+      </div>
+    </div>
+  </section>
+  <style>
+    .p-strip--suru-desktop {
+      background-image:
+        linear-gradient(to bottom right, rgba(210,180,202,0.05) 0%, rgba(210,180,202,0.05) 49.9%, rgba(210,180,202,0) 50%, rgba(210,180,202,0) 100%),
+        linear-gradient(to top right, rgba(74,24,60,0) 0%, rgba(74,24,60,0) 49.9%, rgba(74,24,60,0.34) 50%, rgba(74,24,60,0.34) 100%),
+        linear-gradient(to top right, #f7f7f7 0%, #f7f7f7 49.5%, transparent 50%),
+        linear-gradient(50deg, #E95420 0%, #772953 33%, #2C001E 72%);
+      background-position: left top, right top, left bottom;
+      background-repeat: no-repeat;
+      background-size: 50% 40%, 60% 80%, 102% 20%, 100% 100%;
+      padding-bottom: 6rem;
+      padding-top: 6rem;
+    }
+
+    @media (max-width: 620px) {
+      .p-strip--suru-desktop {
+        background-image:
+          linear-gradient(to bottom right, rgba(210,180,202,0.05) 0%, rgba(210,180,202,0.05) 49.9%, rgba(210,180,202,0) 50%, rgba(210,180,202,0) 100%),
+          linear-gradient(to top right, rgba(74,24,60,0) 0%, rgba(74,24,60,0) 49.9%, rgba(74,24,60,0.34) 50%, rgba(74,24,60,0.34) 100%),
+          linear-gradient(to top right, #f7f7f7 0%, #f7f7f7 49.5%, transparent 50%),
+          linear-gradient(50deg, #E95420 0%, #772953 33%, #2C001E 72%);
+        background-position: left top, right top, left bottom -1px;
+        background-repeat: no-repeat;
+        background-size: 50% 40%, 60% 80%, 102% 20%, 100% 100%;
+      }
+    }
+  </style>
+  <section class="p-strip--light is-deep">
+    <div class="row u-sv3">
+      <h2>What you get when you work with us</h2>
+    </div>
+    <div class="row">
+      <div class="col-8">
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Lorem ipsum dolor sit amet, consectetur ipsum dolor sit amet ipsum dolor sit amet adipiscing elit, sed do eiusmod tempor incididunt</li>
+          <li class="p-list__item is-ticked">Lorem ipsum dolor sit amet, consectetur ipsum dolor sit amet ipsum dolor sit amet adipiscing elit, sed do eiusmod tempor incididunt</li>
+          <li class="p-list__item is-ticked">Lorem ipsum dolor sit amet, consectetur ipsum dolor sit amet ipsum dolor sit amet adipiscing elit, sed do eiusmod tempor incididunt</li>
+          <li class="p-list__item is-ticked">Lorem ipsum dolor sit amet, consectetur ipsum dolor sit amet ipsum dolor sit amet adipiscing elit, sed do eiusmod tempor incididunt</li>
+          <li class="p-list__item is-ticked">Lorem ipsum dolor sit amet, consectetur ipsum dolor sit amet ipsum dolor sit amet adipiscing elit, sed do eiusmod tempor incididunt</li>
+        </ul>
```

Comment:
This is a lot of Lorem Ipsum here. Should we land pages with this content?

Reply:
Don't know...we can hold it off if you think is better.

Reply:
@anthonydillon let's land this page, as I have it captured in the spreadsheet that it still needs some work.

Reply:
Yea ok. Let's hold off updating the copy until it's signed off to save wasting time.